lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4945,6 +4945,9 @@ DefPrimitive('\selectfont', sub {
     my $series = ToString(Expand(T_CS('\f@series')));
     my $shape  = ToString(Expand(T_CS('\f@shape')));
     if    (my $sh = LaTeXML::Common::Font::lookupFontFamily($family)) { MergeFont(%$sh); }
+    elsif ((LookupValue('font')->getEncoding eq 'U') && LoadFontMap($family)) {
+      # Special case hack: If encoding is U, tentatively treat family as the encoding!
```

Comment:
Unsure how safe this is, but probably safe enough for now.

I had seem some missing_file warnings in arXiv with a small single letter `u`, I wonder if they're related?
https://corpora.mathweb.org/corpus/arxmliv/tex%5Fto%5Fhtml/warning/missing%5Ffile/u?all=false